### PR TITLE
Fix broken downloads in Mobile-view

### PR DESCRIPTION
**Pull request by kitallis**
**Story card:** [ch1728](https://app.clubhouse.io/simpledotorg/story/1728/downloads-reports-overdue-don-t-work-on-mobile-view)

Fixes broken downloads on mobile-view:

* Reports

Screenshot 2020-10-27 at 12 52 31 PM

Screenshot 2020-10-27 at 12 52 28 PM

* Overdue

Screenshot 2020-10-27 at 12 48 48 PM

Screenshot 2020-10-27 at 12 49 06 PM

* Misc: remove empty scrollbar showing up on Chrome

image

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -242,7 +242,7 @@ footer {
   .container {
     margin-bottom: 36px;
     white-space: nowrap;
-    overflow-x: scroll;
+    overflow-x: auto;
```
Comment (harimohanraj89) [inline app/assets/stylesheets/application.scss:245]: What's this for?

File: app/views/layouts/overdue.html.erb [hunk 2]
```diff
@@ -14,6 +14,21 @@
         <i class="fas fa-filter"></i> Search filters
       </a>
     </div>
+
+    <section class="mb-4">
```
Comment (harimohanraj89) [inline app/views/layouts/overdue.html.erb:18]: Oh, there's no download link on desktop either right now?

File: app/views/reports/regions/_header.html.erb [hunk 3]
```diff
@@ -95,6 +92,9 @@
           </div>
         <% end %>
       </div>
+      <div class="ml-8px">
+        <%= render "downloads" %>
+      </div>
```
Comment (harimohanraj89) [inline app/views/reports/regions/_header.html.erb:97]: Will this move where the downloads are in desktop?

**Review by harimohanraj89 — APPROVED** [review]
Some questions, but overall looks good.

File: app/views/layouts/overdue.html.erb (hunk 2 above)
Comment (kitallis) [reply app/views/layouts/overdue.html.erb:18]: There is, it's downstairs, under the `desktop nav` section.

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (prabhanshuguptagit) [reply app/assets/stylesheets/application.scss:245]: To remove this weird scrollbar showing up in chrome.

image

File: app/views/reports/regions/_header.html.erb (hunk 3 above)
Comment (kitallis) [reply app/views/reports/regions/_header.html.erb:97]: Posted a screenshot.

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (harimohanraj89) [reply app/assets/stylesheets/application.scss:245]: Ohh that was a scrollbar! Amazing.

**Pull request merged**